ironfish/src/wallet/wallet.ts

```diff
@@ -1355,21 +1359,9 @@ export class Wallet {
     validateAccountImport(accountValue)
 
     let createdAt = accountValue.createdAt
-
-    if (createdAt && !this.nodeClient) {
-      this.logger.debug(
-        `Wallet not connected to node to verify that account createdAt block ${createdAt.hash.toString(
-          'hex',
-        )} (${createdAt.sequence}) in chain. Setting createdAt to null`,
-      )
-      createdAt = null
-    }
-
-    if (createdAt !== null && !(await this.chainHasBlock(createdAt.hash))) {
-      this.logger.debug(
-        `Account ${accountValue.name} createdAt block ${createdAt.hash.toString('hex')} (${
-          createdAt.sequence
-        }) not found in the chain. Setting createdAt to null.`,
+    if (createdAt?.networkId !== this.networkId) {
+      this.logger.warn(
+        `Account ${accountValue.name} networkId ${createdAt?.networkId} does not match wallet networkId ${this.networkId}. Setting createdAt to null.`,
```

Comment:
I think we only want this message when there is a mismatch but it'll show this when there is no birthday. I don't think we need a warning for importing accounts with no birthdays, which will be all older accounts.

Reply:
updated to warn only when networkId is defined